source/includes/_account.md

```diff
@@ -760,18 +782,20 @@ List<Account> accountsByKey = accountApi.searchAccounts(searchKey,
 ```
 
 ```ruby
+account = KillBillClient::Model::Account
+
 search_key = 'John'
 offset = 0
 limit = 100
 with_balance = false
 with_balance_and_cba = false
 
-account.find_in_batches_by_search_key(search_key,
-                                      offset,
-                                      limit,
-                                      with_balance,
-                                      with_balance_and_cba,
-                                      options)
+accounts = account.find_in_batches_by_search_key( search_key,
+                                                  offset,
+                                                  limit,
+                                                  with_balance,
+                                                  with_balance_and_cba,
+                                                  options)
```

Comment:
same comment as above, if there are extra spaces, can you please delete them?

Reply:
Hi Reshma, Yes I added these to align all the values. Below are local render and editor screenshots : 

![image](https://github.com/killbill/slate/assets/75751774/d95632ad-30a9-4672-a284-287fb6c3dbae)
![image](https://github.com/killbill/slate/assets/75751774/506a7aa8-48cf-4b44-9edd-b7307f4cc503)